packages/jest-expo/universal/jest-preset.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const reactNativePreset = require('react-native-web/jest-preset');
+const expoPreset = require('../jest-preset');
+
+const COLORS = {
+  android: 'blueBright',
+  ios: 'white',
+  web: 'magenta',
+  node: 'cyan',
+};
+
+function getModuleFileExtensions(...platforms) {
+  let fileExtensions = [];
+
+  // Support both TypeScript and JavaScript
+  for (const language of ['t', 'j']) {
+    // Support JS and JSX
+    for (const extension of [`${language}s`, `${language}sx`]) {
+      // Ensure order is correct: [platformA.js, platformB.js, js]
+      for (const platform of platforms.concat([''])) {
+        fileExtensions.push([platform, extension].filter(Boolean).join('.'));
+      }
+    }
+  }
+  // Always add this last
+  fileExtensions.push('json');
+  return fileExtensions;
+}
+
+function getPlatformPreset(...platforms) {
+  return {
+    displayName: {
+      name: platforms[0].toUpperCase(),
```

Comment:
Would be nice to display `iOS` and `Android` capitalized the way they are branded